src/main/java/io/fabric8/maven/docker/util/AuthConfigFactory.java

```diff
@@ -669,18 +663,19 @@ private Server checkForServer(Server server, String id, String registry, String
     }
 
     private String decrypt(String password) throws MojoExecutionException {
-        try {
-            // Done by reflection since I have classloader issues otherwise
-            Object secDispatcher = container.lookup(SecDispatcher.ROLE, "maven");
-            Method method = secDispatcher.getClass().getMethod("decrypt",String.class);
-            synchronized(secDispatcher) {
-                return (String) method.invoke(secDispatcher, password);
-            }
-        } catch (ComponentLookupException e) {
-            throw new MojoExecutionException("Error looking security dispatcher",e);
-        } catch (ReflectiveOperationException e) {
-            throw new MojoExecutionException("Cannot decrypt password: " + e.getCause(),e);
-        }
+//        try {
+//            // Done by reflection since I have classloader issues otherwise
+//            Object secDispatcher = container.lookup(SecDispatcher.ROLE, "maven");
+//            Method method = secDispatcher.getClass().getMethod("decrypt",String.class);
+//            synchronized(secDispatcher) {
+//                return (String) method.invoke(secDispatcher, password);
+//            }
+//        } catch (ComponentLookupException e) {
+//            throw new MojoExecutionException("Error looking security dispatcher",e);
+//        } catch (ReflectiveOperationException e) {
+//            throw new MojoExecutionException("Cannot decrypt password: " + e.getCause(),e);
+//        }
```

Comment:
This code is used for decrypting password from maven settings.xml . Why is this commented out? Disabling this might cause problems for users who are using [encryption with maven](https://maven.apache.org/guides/mini/guide-encryption.html)

Reply:
same as above